ksql-functional-tests/src/test/java/io/confluent/ksql/test/rest/RestTestExecutor.java

```diff
@@ -180,8 +180,8 @@ private void produceInputs(final Map<Topic, List<Record>> inputs) {
 
     if (!testCase.expectedError().isPresent()) {
       for (int idx = firstStatic; testCase.getExpectedResponses().size() > idx; ++idx) {
-        final String staticStatement = allStatements.get(firstStatic);
-        final Response staticResponse = testCase.getExpectedResponses().get(firstStatic);
+        final String staticStatement = allStatements.get(idx);
```

Comment:
This assumes that all queries after the first static query are static as well. This assumption is already baked into other parts of `RestTestExecutor`, but if desired I can have this verify the assumption.